package.json

```diff
@@ -34,21 +35,49 @@
     "merge-stream": "^1.0.1",
     "path": ">=0.12.7",
     "semver": "^6.0.0",
-    "stylelint": "^11.1.1",
-    "stylelint-config-wordpress": "^15.0.0",
     "underscore": "^1.9.1"
   },
   "devDependencies": {
-    "nodegit": "^0.24.1"
+    "nodegit": "^0.26.5"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint:css": "wp-scripts lint-style",
+    "lint:js": "wp-scripts lint-js",
+    "format:js": "wp-scripts format-js"
+  },
+  "browserslist": [
+    "last 5 versions",
+    "> 1%",
+    "maintained node versions",
+    "not dead"
+  ],
+  "eslintConfig": {
+    "extends": [
+      "plugin:@wordpress/eslint-plugin/es5"
+    ],
+    "env": {
+      "browser": true,
+      "commonjs": true,
+      "node": false
+    }
+  },
+  "stylelint": {
+    "extends": [
+      "stylelint-config-wordpress/scss"
+    ],
+    "rules": {
+      "no-descending-specificity": null
+    }
   },
   "keywords": [
     "wordpress",
     "wordpress theme",
     "wordpress plugin"
   ],
   "author": "SAU/CAL",
-  "license": "ISC"
+  "license": "ISC",
+  "projectFolders": [
```

Comment:
I'd rather keep project-folders to be a posibility, both for backward compatibility, and for easy upgrade process when updating gulp files (paste in new gulp related files and everything should work).

Reply:
Do you mean that if the entry is not into package.json then search for the project-folders.json file instead?

Reply:
Yep, keep both options, with the file taking precedence over the json option, so that if we paste this new version on an existing project, it just works.